frontend/src/pages/JobFeed/components/TagSelect.tsx

```diff
@@ -19,7 +19,12 @@ type TagSelectProps = {
 export default function TagSelect(props: TagSelectProps) {
   const { onChange, value } = props
   const theme = useTheme()
-  const [{ data }] = useAxios('http://localhost:5000/tags')
+  const [{ data }] = useAxios({
+    url: 'http://localhost:5000/tag',
```

Comment:
based on what stu said today in the meeting, the url should be just `/tag` you dont need the host and port stuff

Reply:
Changed. 
I used the "grep -Rl "localhost" . --exclude-dir={node_modules,build,public}" command, so there should not be any localhost or 127.0.0.1 stuff anymore